docs/development.md

```diff
@@ -805,22 +984,11 @@ GitHub Gist, or other data-sharing platforms such as Dropbox and link them in th
 
  *  For **Python** plugins, we recommend to package your library as a pip module.
     Several excellent ressources, such as [this one](https://packaging.python.org/tutorials/packaging-projects/),
-    explaining this step exist. Please note that it is not necessary that you up-load your package to the
-    Package Index. The crucial part is that you provide the `setup.py`.
+    explaining this step exist. Please note that it is not necessary that you up-load
+    your package to the Package Index. The crucial part is that you provide the `setup.py`.
 
     Create a [gist](https://gist.github.com/) or a GitHub repository, and obtain
```

Comment:
I think you mean command format like this: 
pip install git+https://github.com/{ username }/{ reponame }@{ tag name }#egg={ desired egg name }

Would be good to specify the format and give one example.

Also, I think you cannot host a python module on gist, only github repo will work. So please remove gist here.